### PR TITLE
Added descriptions to advanced

### PR DESCRIPTION
**Pull request by loocool2**
Descriptions of all settings in advanced and their possible uses. Not sure if I should be more specific in some areas.

### CODE REVIEW

File: src/help/guides/advanced.md [hunk 1]
```diff
@@ -4,19 +4,25 @@ lang: en-US
 ---
 
 # Advanced
+The features in this section mostly relates to keeping entries up to date or clearing unnecessary things from the app.
+
 :construction: :construction_worker_man: W.I.P. Description, help us improve through [here](https://github.com/tachiyomiorg/website/edit/master/src/help/guides/advanced.md)!
 
 ## Clear chapter cache
-:construction: :construction_worker_man: W.I.P.
+This setting clears the cache of images from the app. Tachiyomi autoclears the cache when it reaches a certain threshold.
```
Comment (Soitora) [inline src/help/guides/advanced.md]: "automatically clears"?

File: src/help/guides/advanced.md [hunk 2]
```diff
@@ -4,19 +4,25 @@ lang: en-US
...
 
 ## Clear cookies
-:construction: :construction_worker_man: W.I.P.
+
+This setting clears all cookies from Tachiyomi and the webview browser. This may log you out of tracking services and other parts of the app that use cookies to login to the site, such as MangaDex.
```
Comment (Soitora) [inline src/help/guides/advanced.md]: Perhaps we should start capitalizing it as `WebView` just like how Google does it?

File: src/help/guides/advanced.md [hunk 3]
```diff
@@ -4,19 +4,25 @@ lang: en-US
...
 
 ## Clear database
-:construction: :construction_worker_man: W.I.P.
+
+This removes entries of manga that are not in the library. This is useful for solving any problems in a extension that have been fixed but have not been updated in the app yet. 
 
 ## Refresh library metadata
-:construction: :construction_worker_man: W.I.P.
+
+This setting refreshes all aspects of entries in the library to the most recent one that can be found from the source it's from. It updates the cover, grene, description, and the complete status of the entry.
```
Comment (Soitora) [inline src/help/guides/advanced.md]: Genre?

File: src/help/guides/advanced.md [hunk 4]
```diff
@@ -4,19 +4,25 @@ lang: en-US
...
 
 ## Refresh tracking metadata
-:construction: :construction_worker_man: W.I.P.
+
+This setting refreshes all tracking info to the most recent version that can be found on the tracking service, such as completed status, rating, and the last chapter read number.
```
Comment (Soitora) [inline src/help/guides/advanced.md]: "and the number of the latest chapter read." ? "last chapter read number" sounds kind of weird.

**Comment by Soitora** [comment]
Also, like the other guides, can you use a badge to signify the default app value? Otherwise I'll add it afterwards.

File: src/help/guides/advanced.md (hunk 1 above)
Comment (loocool2) [reply src/help/guides/advanced.md]: Sure

File: src/help/guides/advanced.md (hunk 3 above)
Comment (loocool2) [reply src/help/guides/advanced.md]: Yeah, I should really be running this through a spell checker before I open a PR.

File: src/help/guides/advanced.md (hunk 2 above)
Comment (loocool2) [reply src/help/guides/advanced.md]: Sure

File: src/help/guides/advanced.md (hunk 4 above)
Comment (loocool2) [reply src/help/guides/advanced.md]: Sure

**Comment by loocool2** [comment]
There's no default app value here so I didn't add any

**Pull request merged**